### PR TITLE
Update the tools part in roadmap.md

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update tidb roadmap
Follow up: After this PR is merged, @IANTHEREAL will create a separate PR to tick the box.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@IANTHEREAL Could you please involve a technical review(s)?

File: roadmap.md [hunk 1]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
 - [ ] 支持修改列的类型 [#17526](https://github.com/pingcap/tidb/issues/17526)
 - [ ] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
 - [x] 字符集的排序规则支持 `utf8mb4_unicode_ci` 和 `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
-- [ ] TiCDC 兼容 TiDB Binlog 相关的特性，降低用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
-    - [ ] 支持分类更新和插入一行数据事件
-    - [ ] 通过删除或者更新语句修改一行数据时，系统自动记录原始值
-- [ ] 容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
-    - [ ] 支持 MySQL 接收器在上游遇到灾难时将其复制到快照级别的一致状态
+- [ ] 数据导入，Lightning 支持检测出来导入数据中的冲突数据 [#1111](https://github.com/pingcap/br/issues/1111)
```
Comment (TomShawn) [inline roadmap.md:54]: + “检测出来”表示结果
+ 这一行和 Line 55 都是数据导入的话，能否考虑下面这种格式呢？

```
- 数据导入
    - [ ] TiDB Lightning 支持检测导入数据中的冲突数据
    - [ ] TiDB Lightning 支持分布式并行导入
```

File: roadmap.md [hunk 2]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
+- [ ] 数据导入，Lightning 支持分布式并行导入 [#1173](https://github.com/pingcap/br/issues/1173)
```
Comment (TomShawn) [inline roadmap.md:55]:
```suggestion
- [ ] 数据导入：TiDB Lightning 支持分布式并行导入 [#1173](https://github.com/pingcap/br/issues/1173)
```

File: roadmap.md [hunk 3]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
+- [ ] 数据复制，TiCDC 兼容 TiDB Binlog 相关的特性，降低使用 Binlog 用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
```
Comment (TomShawn) [inline roadmap.md:56]: 下面两个子项都勾选了，这一个是否也要勾选呢？
```suggestion
- [x] 数据复制：TiCDC 兼容 TiDB Binlog 相关的特性，降低 TiDB Binlog 用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
```

File: roadmap.md [hunk 4]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
+    - [x] 支持分类更新和插入一行数据事件
+    - [x] 通过删除或者更新语句修改一行数据时，系统自动记录原始值
+- [ ] 数据容灾，容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
```
Comment (TomShawn) [inline roadmap.md:59]:
```suggestion
- [ ] 数据容灾：容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
```

File: roadmap.md [hunk 5]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
+    - [ ] 支持 TiDB 灾备集群在主集群遇到灾难时将备用集群复制到快照级别的一致状态
 - [ ] 支持通过 API 管理 TiCDC [#736](https://github.com/pingcap/ticdc/issues/736)
-- [ ] 支持 `import` SQL 命令 [#18089](https://github.com/pingcap/tidb/issues/18089)
-- [x] 支持 Avro 接收器，使 TiCDC 与 Kafka connect 兼容 [#660](https://github.com/pingcap/ticdc/issues/660)
+- [x] 数据共享，TiCDC 支持 Avro 接收器，使 TiCDC 与 Kafka connect 兼容 [#660](https://github.com/pingcap/ticdc/issues/660)
```
Comment (TomShawn) [inline roadmap.md:62]:
```suggestion
- [x] 数据共享：TiCDC 支持 Avro 接收器，使 TiCDC 与 Kafka connect 兼容 [#660](https://github.com/pingcap/ticdc/issues/660)
```

File: roadmap.md (hunk 1 above)
Comment (IANTHEREAL) [reply roadmap.md:54]: 可以

**Comment by IANTHEREAL** [comment]
@qiancai @TomShawn PTAL, Thanks

File: roadmap.md [hunk 6]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
 - [ ] 支持修改列的类型 [#17526](https://github.com/pingcap/tidb/issues/17526)
 - [ ] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
 - [x] 字符集的排序规则支持 `utf8mb4_unicode_ci` 和 `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
-- [ ] TiCDC 兼容 TiDB Binlog 相关的特性，降低用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
-    - [ ] 支持分类更新和插入一行数据事件
-    - [ ] 通过删除或者更新语句修改一行数据时，系统自动记录原始值
-- [ ] 容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
-    - [ ] 支持 MySQL 接收器在上游遇到灾难时将其复制到快照级别的一致状态
+- 数据导入
```
Comment (TomShawn) [inline roadmap.md:54]:
```suggestion
- [ ] 数据导入
```

File: roadmap.md [hunk 7]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
+    - [ ] TiDB Lightning 支持检测导入数据中的冲突数据
+    - [ ] TiDB Lightning 支持分布式并行导入
+- [ ] 数据复制：TiCDC 兼容 TiDB Binlog 相关的特性，降低 TiDB Binlog 用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
+    - [ ] 支持输出数据到文件系统和云端存储(S3/GCS)
```
Comment (TomShawn) [inline roadmap.md:58]:
```suggestion
- [ ] 数据复制
    - [ ] TiCDC 兼容 TiDB Binlog 相关的特性，降低 TiDB Binlog 用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
    - [ ] 支持输出数据到文件系统和云端存储 (S3/GCS)
```

File: roadmap.md [hunk 8]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
 - [ ] 支持修改列的类型 [#17526](https://github.com/pingcap/tidb/issues/17526)
 - [ ] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
 - [x] 字符集的排序规则支持 `utf8mb4_unicode_ci` 和 `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
-- [ ] TiCDC 兼容 TiDB Binlog 相关的特性，降低用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
-    - [ ] 支持分类更新和插入一行数据事件
-    - [ ] 通过删除或者更新语句修改一行数据时，系统自动记录原始值
-- [ ] 容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
-    - [ ] 支持 MySQL 接收器在上游遇到灾难时将其复制到快照级别的一致状态
+- 数据导入
+    - [ ] TiDB Lightning 支持检测导入数据中的冲突数据
```
Comment (TomShawn) [inline roadmap.md:55]:
```suggestion
    - [ ] TiDB Lightning 支持检测导入数据中的冲突
```

File: roadmap.md [hunk 9]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
+    - [ ] TiDB Lightning 支持分布式并行导入
+- [ ] 数据复制：TiCDC 兼容 TiDB Binlog 相关的特性，降低 TiDB Binlog 用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
+    - [ ] 支持输出数据到文件系统和云端存储(S3/GCS)
+- [ ] 数据容灾：容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
+    - [ ] 支持 TiDB 灾备集群在主集群遇到灾难时将备用集群复制到快照级别的一致状态
```
Comment (TomShawn) [inline roadmap.md:60]:
```suggestion
- [ ] 数据容灾
    - [ ] 容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
    - [ ] 支持 TiDB 灾备集群在主集群遇灾时将备用集群复制到快照级别的一致状态
```

File: roadmap.md (hunk 7 above)
Comment (IANTHEREAL) [reply roadmap.md:58]: `支持输出数据到文件系统和运维存储` 是 `TiCDC 兼容 Binlog ` 的关键子任务，所以这里这么写

File: roadmap.md (hunk 9 above)
Comment (IANTHEREAL) [reply roadmap.md:60]: 这也是一个关键子任务

**Review by TomShawn — APPROVED** [review]
LGTM

File: roadmap.md [hunk 10]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
 - [ ] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
 - [x] 字符集的排序规则支持 `utf8mb4_unicode_ci` 和 `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
-- [ ] TiCDC 兼容 TiDB Binlog 相关的特性，降低用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
-    - [ ] 支持分类更新和插入一行数据事件
-    - [ ] 通过删除或者更新语句修改一行数据时，系统自动记录原始值
-- [ ] 容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
-    - [ ] 支持 MySQL 接收器在上游遇到灾难时将其复制到快照级别的一致状态
+- [ ] 数据导入
+    - [ ] TiDB Lightning 支持检测导入数据中的冲突
+    - [ ] TiDB Lightning 支持分布式并行导入
```
Comment (TomShawn) [inline roadmap.md:56]:
```suggestion
    - [x] TiDB Lightning 支持分布式并行导入
```

File: roadmap.md [hunk 11]
```diff
@@ -51,14 +51,15 @@ aliases: ['/zh/tidb/v5.0/roadmap','/docs-cn/ROADMAP/','/docs-cn/roadmap/','/docs
...
 - [ ] 提升发现及诊断性能问题的效率，降低用户的使用成本 [#18867](https://github.com/pingcap/tidb/issues/18867)
 - [x] 字符集的排序规则支持 `utf8mb4_unicode_ci` 和 `utf8_unicode_ci` [#17596](https://github.com/pingcap/tidb/issues/17596)
-- [ ] TiCDC 兼容 TiDB Binlog 相关的特性，降低用户使用 TiCDC 的成本 [#690](https://github.com/pingcap/ticdc/issues/690)
-    - [ ] 支持分类更新和插入一行数据事件
-    - [ ] 通过删除或者更新语句修改一行数据时，系统自动记录原始值
-- [ ] 容灾中支持快照级别的一致性数据复制 [#691](https://github.com/pingcap/ticdc/issues/691)
-    - [ ] 支持 MySQL 接收器在上游遇到灾难时将其复制到快照级别的一致状态
+- [ ] 数据导入
+    - [ ] TiDB Lightning 支持检测导入数据中的冲突
+    - [x] TiDB Lightning 支持分布式并行导入
```
Comment (TomShawn) [inline roadmap.md:56]:
```suggestion
    - [ ] TiDB Lightning 支持分布式并行导入
```

**Comment by TomShawn** [comment]
@IANTHEREAL Only the master branch has the TiDB roadmap doc. So no cherry-pick is needed.

**Pull request merged**